packages/ui/src/theme/css/component/toggleButton.scss

```diff
@@ -0,0 +1,128 @@
+%togglebutton-shared {
+  border-color: var(--amplify-components-togglebutton-border-color);
+  color: var(--amplify-components-togglebutton-color);
+}
+
+.amplify-togglebutton {
+  @extend %togglebutton-shared;
+
+  &:hover {
+    @extend %togglebutton-shared;
```

Comment:
I presume this is needed to override default button styling? If so, can we add a comment?

Reply:
Right, let me add a comment